typings/index.d.ts

```diff
@@ -0,0 +1,101 @@
+declare module '@paralleldrive/react-feature-toggles' {
+  import * as React from 'react';
+
+  interface Feature {
+    readonly name: string;
+    readonly isActive: boolean;
+    readonly dependencies?: ReadonlyArray<string>;
+  }
+
+  interface Query {
+    readonly ft: string;
+  }
+
+  interface Req {
+    readonly query: Query;
+  }
+
+  interface CurrentActiveFeatures {
+    readonly initialFeatures: ReadonlyArray<string>;
+    readonly req?: Req;
+    readonly search?: string;
+  }
+
+  function Feature(
```

Comment:
Have you tested this? I think this should be taking a props object, not individual arguments correct? Make sure this works for both the standard way and the render-prop way, probably by specifying overloads

Reply:
I have tested this with both and it does work, but I see what you are saying about it taking the props object and not the arguments. I think I updated it with how it should be implemented now.

Reply:
Nice, much better